localization/english/culture/ctp_titles_l_english.yml

```diff
@@ -414,6 +414,144 @@
+ duke_feudalclan_male_cisalpine:0 "Dux"
+ duke_feudalclan_female_cisalpine:0 "Duchissa"
+ duchy_feudalclan_cisalpine:0 "Ducato"
+ duke_feudalclan_male_cisalpine_independent:0 "Prencep"
+ duke_feudalclan_female_cisalpine_independent:0 "Prencepessa"
+ duchy_feudalclan_cisalpine_independent:0 "Prencepato"
+ title_prince_male_cisalpine:0 "Prencep"
+ title_prince_female_cisalpine:0 "Prencepessa"
+ king_male_cisalpine:0 "Rex"
+ king_female_cisalpine:0 "Regina"
+ kingdom_cisalpine:0 "Regno"
+ emperor_male_cisalpine:0 "Imperator"
+ emperor_female_cisalpine:0 "Imperatrice"
+ empire_cisalpine:0 "Imperio"
+ ##Sardinian##
+ baron_male_sardinian:0 "Baròni"
+ baron_female_sardinian:0 "Barònissa"
+ barony_sardinian:0 "Baronia"
+ count_feudalclan_male_sardinian:0 "Conti"
+ count_feudalclan_female_sardinian:0 "Contissa"
+ county_feudalclan_sardinian:0 "Contadu"
+ count_feudalclan_male_sardinian_independent:0 "Judike"
+ count_feudalclan_female_sardinian_independent:0 "Judike"
+ county_feudalclan_sardinian_independent:0 "Judicadu"
+ duke_feudalclan_male_sardinian:0 "Duca"
+ duke_feudalclan_female_sardinian:0 "Duchissa"
+ duchy_feudalclan_sardinian:0 "Ducatu"
+ duke_feudalclan_male_sardinian_independent:0 "Judike"
+ duke_feudalclan_female_sardinian_independent:0 "Judike"
+ duchy_feudalclan_sardinian_independent:0 "Judicadu"
+ title_prince_male_sardinian:0 "Príncipi"
+ title_prince_female_sardinian:0 "Printzèsa"
+ king_male_sardinian:0 "Rey"
+ king_female_sardinian:0 "Reína"
+ kingdom_sardinian:0 "Rennu"
+ emperor_male_sardinian:0 "Imperatori"
+ emperor_female_sardinian:0 "Imperatrice"
+ empire_sardinian:0 "Imperu"
+ ##Sicilian##
+ baron_male_sicilian:0 "Baruni"
+ baron_female_sicilian:0 "Barunissa"
+ barony_sicilian:0 "Barunia"
+ count_feudalclan_male_sicilian:0 "Conti"
+ count_feudalclan_female_sicilian:0 "Contissa"
+ county_feudalclan_sicilian:0 "Contadu"
+ count_feudalclan_male_sicilian_independent:0 "Principi"
+ count_feudalclan_female_sicilian_independent:0 "Principissa"
+ county_feudalclan_sicilian_independent:0 "Principatu"
+ duke_feudalclan_male_sicilian:0 "Duca"
+ duke_feudalclan_female_sicilian:0 "Duchissa"
+ duchy_feudalclan_sicilian:0 "Ducatu"
+ duke_feudalclan_male_sicilian_independent:0 "Principi"
+ duke_feudalclan_female_sicilian_independent:0 "Principissa"
+ duchy_feudalclan_sicilian_independent:0 "Principatu"
+ title_prince_male_sicilian:0 "Principi"
+ title_prince_female_sicilian:0 "Principissa"
+ king_male_sicilian:0 "Re"
+ king_female_sicilian:0 "Regina"
+ kingdom_sicilian:0 "Regnu"
+ emperor_male_sicilian:0 "Imperaturi"
+ emperor_female_sicilian:0 "Imperatrici"
+ empire_sicilian:0 "Impèriu"
+ ##Lombard##
+ baron_male_lombard:0 "Baro"
+ baron_female_lombard:0 "Baronissa"
+ barony_lombard:0 "Baronia"
+ count_feudalclan_male_lombard:0 "Gastaldus"
+ count_feudalclan_female_lombard:0 "Gastaldissa"
+ county_feudalclan_lombard:0 "Gastaldia"
+ count_feudalclan_male_lombard_independent:0 "Princeps"
```

Comment:
Am I the only one who's weirded out by Latin titles for Lombard?

Reply:
I believe lombard is dead and not used save for historic rulers.

Reply:
Nope. I just went through some of province files. Everything in the "d_spoleto" duchy has the lombard culture for 867 start.

Reply:
Then we should double-think it probably - I can't believe they retained proper declensions of Latin.

Reply:
Lombard is as good as non-existent now